packages/web-app-password-protected-folders/src/composables/useCreateFileHandler.ts

```diff
@@ -26,20 +27,22 @@ export const useCreateFileHandler = () => {
     }
 
     const folderPath = '/.' + fileName
+    const folder = await clientService.webdav.createFolder(personalSpace, { path: folderPath })
```

Comment:
does this mean if a protected folder is created in any project space, the folder (hidden folder) will be created in the creator's personal space?

Reply:
Also if I create `test1` folder in my personal space. and try to create `test1` in project space, it doesn't allow creation of `test1` in the project space.

Reply:
Done

Reply:
@LukasHirt 
should the user be able to create the same name protected folder in the project spaces?
Suppose, I have manually created `.test` folder in my personal space, and now I want to create a protected folder `test`  in the project space. Should this be allowed? Currently, it doesn't. Wonder if this is expected behavior.

![Screenshot from 2025-02-11 17-00-52](https://github.com/user-attachments/assets/0f2bc491-87ff-4c61-a3c3-2da67d021524)

Reply:
If expected then looks good :+1:

Reply:
It is not optimal solution but I think in this case it would be fine to have this limitation and simply force the user to rename it. The hidden folder name needs to currently match the .psec file name as it is used when deleting the file and folder together. Meaning adding some suffix to the folder name would break the deletion then. So in favor of keeping this feature technically simple, I would say forcing the user to have unique name is fine. What do you say @kobergj

Reply:
Isn't the folder in the personal space under `.PasswordProtectedFolders/projects/<space_name>`? So that would mean a user needs to create the exact same folder structure (including the space name) to achieve a name clash?

Reply:
🤦 and of course I missed this requirement... you are right. I will look into this asap.

Reply:
hmm, I thought something changed internally in the requirements and I didn't ask about it while reviewing. should have just asked it.

Reply:
Fixed